test/unit/test_diskfile.py

```diff
@@ -237,23 +238,33 @@ def test_app_iter_ranges(self, mock_mri):
 class TestDiskFile(unittest.TestCase):
     """Tests for swift_scality_backend.diskfile.DiskFile"""
 
+    @staticmethod
+    def hash_str(*strings):
+        "Get a SHA1 hex hash of the `str` passed as arguments"
+
+        sha1 = hashlib.sha1()
+        map(sha1.update, strings)
```

Comment:
Same `map` comment. Also, instead of `*strings`, why not use `strings` and require to pass in an iterable? Seems like a better suited API to me.